### PR TITLE
Restructure features to minimize the testing data creation

### PR DESCRIPTION
**Pull request by RichardXuan**
Restructure features to minimize the testing data creation

### CODE REVIEW

File: features/steps/common_steps.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def step_impl(context):
 
 @given('I go to an embargoed flaw detail page')
 def step_impl(context):
-    go_to_specific_flaw_detail_page(context.browser, True)
+    go_to_specific_flaw_detail_page(context.browser)
```
Comment (huiwangredhat) [inline features/steps/common_steps.py:36]: This line is the same with line 31?

File: features/utils.py [hunk 2]
```diff
@@ -122,15 +123,25 @@ def go_to_home_page(browser):
...
+def go_to_specific_flaw_detail_page(browser):
     """
     Go to a specific flaw detail page
     """
-    go_to_home_page(browser)
-    home_page = HomePage(browser)
-    home_page.quickSearchBox.clear_text()
-    home_page.quickSearchBox.set_text(cve_id)
-    home_page.click_btn('quickSearchBtn')
+    cve_id = os.getenv("FLAW_ID")
```
Comment (huiwangredhat) [inline features/utils.py:130]: How to get the FLAW_ID? export the FlAW_ID after finishing the flaw_create_*.feature?

File: features/steps/common_steps.py (hunk 1 above)
Comment (RichardXuan) [reply features/steps/common_steps.py:36]: Yes.

File: features/utils.py (hunk 2 above)
Comment (RichardXuan) [reply features/utils.py:130]: Check https://github.com/RedHatProductSecurity/osim/blob/OSIDB-2692/features/steps/flaw_create.py#L135 and https://github.com/RedHatProductSecurity/osim/blob/OSIDB-2692/features/steps/flaw_create.py#L85 .
If you want run a single case locally, just export `FlAW_ID` you want operate.

Comment (huiwangredhat) [reply features/utils.py:130]: I think if the create flaw function has something wrong, it will block all the following testing, right?

Comment (RichardXuan) [reply features/utils.py:130]: Yes. It will.

**Review by huiwangredhat — APPROVED** [review]

**Pull request merged**